ffi/src/handle.rs

```diff
@@ -547,7 +547,8 @@ mod tests {
     pub struct MutNotSync;
 
     // Because tests compile as binaries against packages, this test can only run correctly if we
-    // use the `internal-api` feature to make mod handle public. Otherwise it's inaccessible for testing
+    // use the `internal-api` feature to make mod handle public. Otherwise it's inaccessible for
+    // testing
     #[test]
     #[cfg(feature = "internal-api")]
```

Comment:
```suggestion
    #[internal-api]
```
?

Or are we intentionally not marking this as an "official" internal API, and only expose it for testing?

Reply:
this was just to avoid pulling in delta_kernel_derive to FFI crate - I think this would have been the only callsite. happy to add though, WDYT

Reply:
Ack. Agree we should keep as-is.